### PR TITLE
Feature/network status symbol

### PR DESCRIPTION
**Pull request by Dzejkop**

### CODE REVIEW

File: patchkit-launcher-qt-src/src/launcherworker.cpp [hunk 1]
```diff
@@ -321,11 +321,11 @@ void LauncherWorker::updatePatcher(const Data& t_data)
     }
 }
 
-void LauncherWorker::startPatcher(const Data& t_data)
+void LauncherWorker::startPatcher(const Data& t_data, bool isOfline)
```
Comment (genail) [inline patchkit-launcher-qt-src/src/launcherworker.cpp]: should be `isOffline`

File: patchkit-launcher-qt-app/src/launcher.cpp [hunk 2]
```diff
@@ -51,7 +51,7 @@ void Launcher::onError(DownloadError t_error)
                 emit m_state.respond();
 
                 m_worker.stop();
-                m_worker.startPatcher();
+                m_worker.startPatcher(true);
```
Comment (witcher112) [inline patchkit-launcher-qt-app/src/launcher.cpp]: `true` is very misguiding here. I'd suggest creating and using enum here.

File: patchkit-launcher-qt-src/src/launcherworker.h [hunk 3]
```diff
@@ -35,7 +35,7 @@ class LauncherWorker : public QThread
     void cancel();
 
     bool canStartPatcher() const;
-    void startPatcher();
+    void startPatcher(bool isOffline = false);
```
Comment (witcher112) [inline patchkit-launcher-qt-src/src/launcherworker.h]: Avoid default parameters, it is very important one and it should be always explicitly specified.

File: patchkit-launcher-qt-src/src/launcherworker.h [hunk 4]
```diff
@@ -59,7 +59,7 @@ private slots:
 
     bool isLocalPatcherInstalled() const;
     void updatePatcher(const Data& t_data);
-    void startPatcher(const Data& t_data);
+    void startPatcher(const Data& t_data, bool isOffline = false);
```
Comment (witcher112) [inline patchkit-launcher-qt-src/src/launcherworker.h]: Avoid default parameters, it is very important one and it should be always explicitly specified.

File: patchkit-launcher-qt-src/src/localpatcherdata.cpp [hunk 5]
```diff
@@ -136,7 +136,7 @@ void LocalPatcherData::install(const QString& t_downloadedPath, const Data& t_da
     IOUtils::writeTextToFile(locations.patcherIdInfoFilePath(), getPatcherId(t_data));
 }
 
-void LocalPatcherData::start(const Data& t_data)
+void LocalPatcherData::start(const Data& t_data, bool isOnline)
```
Comment (witcher112) [inline patchkit-launcher-qt-src/src/localpatcherdata.cpp]: Another proof that enum would be better - mostly you are using `isOffline` and here we can see `isOnline` which seems not consistent.

**Pull request merged**